force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -321,14 +406,18 @@ public with sharing class ClassesObjectsMethods {
         List<String> results = new List<String>();
         // Create a new instance of Person class
         // Name the person 'John Doe' and set the age to 28
+        Person person1 = new Person('John Doe', 28);
 
         // Add the details of the person using the introduceYourself method
+         results.add(person1.introduceYourself());
 
         // Get the name of the person using the getter method and add it to the list
+        results.add(person1.getName());
 
         // Get the age of the person using the getter method and add it to the list
+        results.add(String.valueOf(person1.getAge()));
         
-        return null; // Replace null with the variable you used to store the result
+        return results; // Replace null with the variable you used to store the result
```

Comment:
good work here. An alternative is to add these values directly to the List constructor.  But I think it might be less readable in this case. 

```
 public static List<String> personDetails() {
        Person person = new Person('John Doe', 28);
        return  new List<String>{person.introduceYourself(), person.getName(), String.valueOf(person.getAge())};
    }
 ```